### ISSUE TITLE
Feature request: Recognize and embed more video links

### ISSUE THREAD

**Issue by sevonj** [Xyphyn/photon#367]
- [x] m.youtube.com #366 
- [x] youtube.com/live #370 
- [ ] Invidious
- [ ] PeerTube
- [ ] Piped
- [ ] odysee.com

...and probably a lot more I couldn't think of right now.

**Comment by Xyphyn** [comment]
The only issue with this is that I can't recognize every invidious/piped instance since they don't have a global url schema. I could just have a giant list of all the piped and invidious instances, or possibly add a button to tell photon to recognize that as an invidious instance.

**Comment by sevonj** [comment]
Yeah, I thought this might be the case for decentralized services.

**Comment by sevonj** [comment]
Not making a commit that touches YT regex again while the previous PR is up, but just adding the word live makes youtube streams work.

Example post: https://phtn.app/post/lemm.ee/37729007

**Comment by sevonj** [comment]
I'm adding Odysee to the list as I think it's the largest Youtube alternative out there.

**Comment by Krafting** [comment]
Now that Loops is out, it would be awesome to have them integrated in the Photon UI 

Tesseract (another UI) integrate them!  ( https://p.lemmy.world/post/lemmy.world/23390364 )
